spec.html

```diff
@@ -28089,13 +28255,20 @@ <h1>Static Semantics: ImportEntries ( ): a List of ImportEntry Records</h1>
         </emu-alg>
         <emu-grammar>ImportDeclaration : `import` ImportClause FromClause `;`</emu-grammar>
         <emu-alg>
-          1. Let _module_ be the sole element of ModuleRequests of |FromClause|.
+          1. Let _module_ be the SV of |FromClause|.
           1. Return ImportEntriesForModule of |ImportClause| with argument _module_.
         </emu-alg>
         <emu-grammar>ImportDeclaration : `import` ModuleSpecifier `;`</emu-grammar>
         <emu-alg>
           1. Return a new empty List.
         </emu-alg>
+        <emu-grammar>ImportDeclaration : `import` `source` ImportedBinding FromClause `;`</emu-grammar>
+        <emu-alg>
+          1. Let _module_ be the SV of |FromClause|.
+          1. Let _localName_ be the sole element of BoundNames of |ImportedBinding|.
+          1. Let _entry_ be the ImportEntry Record { [[ModuleRequest]]: _module_, [[ImportName]]: ~source~, [[LocalName]]: _localName_ }.
```

Comment:
The [description of ImportEntry Records](https://tc39.es/ecma262/multipage/ecmascript-language-scripts-and-modules.html#importentry-record) will need to be updated to reflect the new possible value for [[ImportName]], and its interpretation.

Reply:
Added in https://github.com/tc39/proposal-source-phase-imports/pull/52.